### PR TITLE
Fauna integration

### PR DESCRIPTION
**Pull request by nspilman**
Reopening PR, as previous PR was merging with Master.
Resource page now reads to Fauna Db. Once this is merged with Dev, Dev will be ready to PR with master.

### CODE REVIEW

File: src/components/Resources/Form.js [hunk 1]
```diff
@@ -6,15 +6,24 @@ import Message from "./Message";
 
 import "./Form.css";
 
+const stringFormattedDateTime = () =>{
+    const now = new Date();
+        const month = now.getMonth()+1
```
Comment (mrdanielmax) [inline src/components/Resources/Form.js]: Mind adding a comment on why we increment getMonth() by 1, but not the other date values (ie day or year)? Also, is the indentation and sporadic use semi-colons intentional/part of faunaDB?

File: src/components/Resources/index.js [hunk 2]
```diff
@@ -20,32 +20,22 @@ class Resources extends Component {
...
+            const resp = await axios.get("/.netlify/functions/allResources");
+            const resources = await resp.data.map(resp => resp.data);
+
             const categories = [
-                ...new Set(resources.map(resource => resource["language/topic"]))
+                ...new Set(resources.map(resource => resource.topic))
             ];
             this.setState({ resources });
             this.setState({ categories });
+            // console.log(resources)
```
Comment (mrdanielmax) [inline src/components/Resources/index.js]: rm

File: src/components/Resources/index.js [hunk 3]
```diff
@@ -20,32 +20,22 @@ class Resources extends Component {
...
         };
         resourceData();
     }
     
     render() {
         const { resources, categories } = this.state;
+        console.log(resources);
```
Comment (mrdanielmax) [inline src/components/Resources/index.js]: any point of merging console logs?

File: functions/allResources/delete.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ exports.handler = async (event, context) => {
   const id = event.id
   console.log(`Function 'delete' invoked. delete id: ${id}`)
   return client
-    .query(q.Delete(q.Ref(`classes/items/${id}`)))
+    .query(q.Delete(q.Ref(`classes/Resource/${id}`)))
```
Comment (mrdanielmax) [inline functions/allResources/delete.js]: are capitol class/resources names convention for this project? Seems as though the old one was all lower-case

File: src/components/Resources/Form.js (hunk 1 above)
Comment (nspilman) [reply src/components/Resources/Form.js]: Added comment And fixed indentation.  getMonth() indexes from 0, from 0 to 11. We add 1 to make it 1 to 12.

File: src/components/Resources/index.js (hunk 3 above)
Comment (nspilman) [reply src/components/Resources/index.js]: deleted

**Pull request merged**